### PR TITLE
Polishing repository

### PR DESCRIPTION
**Pull request by ilfa**
- docs
- validation on real API
- add demo apiKey for Swagger UI

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
```
Comment (TheUnderScorer) [inline .github/workflows/validate.yml:1]: This workflow looks quite similar to the "tests.yml" one. Maybe we could merge them together?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The repository contains [Fingerprint Pro Server API](https://dev.fingerprint.com/docs/server-api) definition in OpenAPI format.
+
+## Schema
+
+You can find the definition in the [schemes](/schemes) folder. You can find information about OpenAPI 3.0.3 in the [official specification](https://spec.openapis.org/oas/v3.0.3).
+
+## Demo app
+
+Code of demo application you can find in [src](src) folder. It uses [Swagger UI](https://github.com/swagger-api/swagger-ui) to render schema and provide sandbox.
```
Comment (TheUnderScorer) [inline CONTRIBUTING.md:11]: How about:

```suggestion
Code of demo application can be found in [src](src) folder. It uses [Swagger UI](https://github.com/swagger-api/swagger-ui) to render schema and provide sandbox.
```

File: bin/validateSchema.js [hunk 3]
```diff
@@ -22,25 +26,81 @@ ajv.addFormat('date-time', {
...
+    },
+    {
+      name: 'Limit 500',
+      path: './examples/visits_limit_500.json',
+    },
+  ];
+
+  return mocks.map(({ name, path }) => ({
+    name: `${name} mock`,
+    jsonData: JSON.parse(fs.readFileSync(path).toString()),
```
Comment (TheUnderScorer) [inline bin/validateSchema.js:49]: Maybe we could just do:
```suggestion
    jsonData: require(path),
```
?

Comment (ilfa) [reply bin/validateSchema.js:49]: Not for ES Module, can replace for it, but not sure it is simpler (a lot of things became async plus to the snippet):

```suggestion
     jsonData: (
        await import(path, {
          assert: {
            type: 'json',
          },
        })
      ).default,
```

Comment (TheUnderScorer) [reply bin/validateSchema.js:59]: Ah, good point, I've forgot that this is not a ES Module!

File: .github/workflows/validate.yml (hunk 1 above)
Comment (ilfa) [reply .github/workflows/validate.yml:1]: Done!

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**